plotly/tools.py

```diff
@@ -95,28 +61,30 @@ def get_config_defaults():
             # do something
 
     """
-    return dict(_FILE_CONTENT[CONFIG_FILE])  # performs a shallow copy
-
-
-def check_file_permissions():
-    return _file_permissions
+    return dict(FILE_CONTENT[CONFIG_FILE])  # performs a shallow copy
 
 
 def ensure_local_plotly_files():
     """Ensure that filesystem is setup/filled out in a valid way"""
-    if _file_permissions:
+    if check_file_permissions():
         for fn in [CREDENTIALS_FILE, CONFIG_FILE]:
             utils.ensure_file_exists(fn)
             contents = utils.load_json_dict(fn)
-            for key, val in list(_FILE_CONTENT[fn].items()):
+            for key, val in list(FILE_CONTENT[fn].items()):
                 # TODO: removed type checking below, may want to revisit
                 if key not in contents:
                     contents[key] = val
             contents_keys = list(contents.keys())
             for key in contents_keys:
-                if key not in _FILE_CONTENT[fn]:
+                if key not in FILE_CONTENT[fn]:
                     del contents[key]
             utils.save_json_dict(fn, contents)
+
+        # make a request to get graph reference if DNE.
```

Comment:
but it will always exist right, since we ship with a version? In the future, this will change to `# make a request to get graph reference if outdated` presumably?